### PR TITLE
Add default dashboard and widgets

### PR DESCRIPTION
**Pull request by orius123**
Adding folder for default dashboards and widgets

### CODE REVIEW

File: Dashboards/dashboard-Incidents.json [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+				},
+				"params": {
+					"colors": {
+						"isEnabled": true,
+						"items": {
+							"#00CD33": {
+								"value": 0
+							},
+							"#FAC100": {
+								"value": 0
```
Comment (Bars92) [inline Dashboards/dashboard-Incidents.json]: Orange should be 1, (already got 0 as green).

File: Widgets/widget-CommonValuePerField.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (Bars92) [inline Widgets/widget-CommonValuePerField.json]: remove widget.

File: Dashboards/dashboard-Incidents.json (hunk 1 above)
Comment (meirwah) [reply Dashboards/dashboard-Incidents.json]: @orius123 no need to fix this?

Comment (orius123) [reply Dashboards/dashboard-Incidents.json]: Nope

**Pull request merged**